test/jcpsantiago/bulgogi_test.clj

```diff
@@ -1,9 +1,90 @@
 (ns jcpsantiago.bulgogi-test
   (:require
     [clojure.test :refer :all]
-    [jcpsantiago.bulgogi :refer :all]))
+    [jcpsantiago.bulgogi :as SUT]
```

Comment:
just curious how did you come up with SUT?

Reply:
I've been using it for a few years, [SUT](https://en.wikipedia.org/wiki/System_under_test).
I like being able to immediately spot which parts of the test are the thing we are actually testing. Having the same naming helps and the upper case makes it easy to spot.

Reply:
cool, I wasn't aware of this 👍 sounds like a good concept indeed